README.md

```diff
@@ -202,12 +176,6 @@ You can deploy this project using:
 - GitHub: [@ijosue16](https://github.com/ijosue16)
 
 
-**👤 Author2**
-
-- GitHub: [@maccony202](https://github.com/maccony202)
-
-
-
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] OPTIONAL
Kindly ensure that the _Future Features_ section on _line 184_ of your ReadMe includes 1-3 potential features to be added later.